tests/unit/argument-test.js

```diff
@@ -22,7 +22,6 @@ test('it works', function(assert) {
   assert.equal(foo.get('baz'), 2, 'class field default gets set correctly');
 
   assert.equal(fooWithValues.get('bar'), 3, 'argument default can be overriden');
-  assert.equal(fooWithValues.get('baz'), 2, 'class field default cannot be overriden');
```

Comment:
Because this behavior is different in Ember `3.6.0`, I decided to remove this and a few other assertions that were dealing with class properties not being overwritten by arguments to the object.

It seems to me like those assertions were all testing the Ember behavior, _not_ the library behavior, and thus were susceptible to breaking when the upstream behavior changed.